homeassistant/components/proxmoxve/__init__.py

```diff
@@ -58,6 +59,9 @@
                                         vol.Optional(CONF_CONTAINERS, default=[]): [
                                             cv.positive_int
                                         ],
+                                        vol.Optional(CONF_STORAGE, default=[]): [
```

Comment:
We no longer accept changes to the YAML configuration of integrations that connect to a device or service. This integration needs to implement a UI configuration flow first.

See: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

For more information to get started on implementing this, see: <https://developers.home-assistant.io>

👍